src/portable/raspberrypi/rp2040/hcd_rp2040.c

```diff
@@ -98,12 +98,21 @@ static void __no_inline_not_in_flash_func(hw_xfer_complete)(struct hw_endpoint *
     uint8_t ep_addr = ep->ep_addr;
     uint xferred_len = ep->xferred_len;
     hw_endpoint_reset_transfer(ep);
+
+    // If this is an interrupt endpoint disable it until host stack requests another
+    // transfer
+    if (ep->transfer_type == TUSB_XFER_INTERRUPT) {
+        usb_hw_clear->int_ep_ctrl = (1 << (ep->interrupt_num + 1));
```

Comment:
I have an question though, according to the rp2040 DS, if the buffer is `not full and available`, there won't be any transfer. This seems to indicate we have an un-scheduled transfer occurred. 

![image](https://user-images.githubusercontent.com/249515/174113220-ab5506be-3355-4c30-9c21-51ecc84c1c58.png)

Reply:
Good point. I implemented this a while ago and rebased it, so maybe it is no longer necessary. In a comment here https://github.com/liamfraser/tinyusb/commit/a38b54d1f2d2b3e756eea475f7114f888ad960fc#commitcomment-68710466 you mention there was an issue with queueing multiple transfers that is now fixed. I remember seeing a race condition at the time so that could have fixed it.

Reply:
@liamfraser would you mind re-testing it with this specific command along its enabled counterpart when starting new transfer to see if the issue still occurs.

Reply:
Ah I remembered that bug, it occurs when plugging hub with already-attached multiple devices to rp2040 host and cause multiple control conflicts. It should be resolved, maybe we should revert this change ?

Reply:
I will test it now without the interrupt endpoint change